scalanet/src/io/iohk/scalanet/peergroup/kademlia/KRouter.scala

```diff
@@ -419,14 +418,16 @@ object KRouter {
     *          also bucket maximum size. In paper mentioned as replication parameter
     * @param serverBufferSize maximum size of server messages buffer
     * @param refreshRate frequency of kademlia refresh procedure
+    * @param parallelism level of parallelism the request will be processed
```

Comment:
I would change this comment to something more infromative for potential developer, maybe something like  `number of concurrent tasks which will handle incoming requests` ? (or similar)